scripts/tess/tess.py

```diff
@@ -0,0 +1,178 @@
+       author = {{Caldwell}, Douglas A. and {Tenenbaum}, Peter and {Twicken}, Joseph D. and {Jenkins}, Jon M. and {Ting}, Eric and {Smith}, Jeffrey C. and {Hedges}, Christina and {Fausnaugh}, Michael M. and {Rose}, Mark and {Burke}, Christopher},
+        title = "{TESS Science Processing Operations Center FFI Target List Products}",
+      journal = {Research Notes of the American Astronomical Society},
+     keywords = {Catalogs, CCD photometry, Stellar photometry, 205, 208, 1620, Astrophysics - Earth and Planetary Astrophysics, Astrophysics - Instrumentation and Methods for Astrophysics, Astrophysics - Solar and Stellar Astrophysics},
+         year = 2020,
+        month = nov,
+       volume = {4},
+       number = {11},
+          eid = {201},
+        pages = {201},
+          doi = {10.3847/2515-5172/abc9b3},
+archivePrefix = {arXiv},
+       eprint = {2011.05495},
+ primaryClass = {astro-ph.EP},
+       adsurl = {https://ui.adsabs.harvard.edu/abs/2020RNAAS...4..201C},
+      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+TESS Light Curves From Full Frame Images ("TESS-SPOC")
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://archive.stsci.edu/hlsp/tess-spoc"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY 4.0"
+
+_VERSION = "0.0.1"
+
+# Full list of features available here:
+# https://archive.stsci.edu/files/live/sites/mast/files/home/missions-and-data/active-missions/tess/_documents/EXP-TESS-ARC-ICD-TM-0014-Rev-F.pdf
+# https://cdsarc.cds.unistra.fr/ftp/IV/39/ReadMe
+#TODO: add additional features from the TIC catalog
+
+_FLOAT_FEATURES = [
+    "RA",
+]
+
+# Features that correspond to fluxes
+#_FLUX_FEATURES = [
+#    "FLUX",
+#    "FLUX_ERR",
+#]
+
+#_BOOL_FEATURES = [
+#    "ZWARNING"
+#]
+
+
+class TESS(datasets.GeneratorBasedBuilder):
+    """TESS Light Curves From Full Frame Images from the TESS Science Processing Operations Center ("TESS-SPOC")"""
+
+    VERSION = _VERSION
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="all",
+            version=VERSION,
+            data_files=DataFilesPatternsDict.from_patterns(
+                {"train": ["*/healpix=*/*.hdf5"]}
+            ),
+            description="TESS-SPOC light curves (S0064)",
+        )
+    ]
+
+    DEFAULT_CONFIG_NAME = "all"
+
+    @classmethod
+    def _info(self):
+        """Defines the features available in this dataset."""
+        # Starting with all features common to time series datasets
+        features = {
+            "lightcurve": Sequence({
+                "time":  Value(dtype="float32"),
+                "flux": Value(dtype="float32"),
+                "flux_err": Value(dtype="float32"),
+                #"quality_mask": Value(dtype="int32"),
+                #"band": Value('string'),
```

Comment:
did you intentionally comment out these quantities?

Reply:
Yes! Directly subtracted the quality mask from the start to make things simpler.